### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by kingqabil**
Modified the Awesome books application to have:

A Navigation bar.
Three content sections:
 a. Books list.
 b. Add book form.
 c. Contact info.
Used this wireframe as a guide:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,62 @@
...
+    <div class="container d-flex">
+      <section id="div-contact">
+        <h2 class="contact-info">Contact Information</h2>
+          <ul class="contactInfo d-none">
+              <li><span>Do you have any questions or you want to reach out to say 'Hello'?</span>You can reach out to us!</li>
+              <li><span>Our e-mail:</span> johndoe@gmail.com</li>
+              <li><span>Our phone number:</span> 004358645378</li>
+              <li><span>Our address:</span> Microverse</li>
+          </ul>
+      </section>
```
Comment (Dipeshtwis) [inline index.html]: - [ ] Please keep all the content in the center to follow the design. For now, it's toward the left and not looking good.

### Your design

Screenshot from 2021-11-25 19-49-33

### Original design

Screenshot from 2021-11-25 19-53-30

File: index.html [hunk 2]
```diff
@@ -1,24 +1,62 @@
...
-    <main id="main">
       <div class="books"></div>
+    </div>
+    <section class="input-data">
+      <h3>Add A Book</h3>
       <form method="get">
         <input id="title" type="text" placeholder="Title" required><br><br>
         <input id="author" type="text" placeholder="Author" required><br><br>
         <button type="submit" id="add">Add</button>
       </form>
```
Comment (Dipeshtwis) [inline index.html]: - [ ] Please make sure to follow the design of the input field and the add button. Please fix that.

### Your design

Screenshot from 2021-11-25 19-51-28


### Original desig

Screenshot from 2021-11-25 19-53-23
n

File: index.js [hunk 3]
```diff
@@ -55,9 +54,68 @@ addBtn.addEventListener('click', (e) => {
...
+const contactPage = document.querySelector('body > main > div.container.d-flex');
+const myTime = document.querySelector('body > main > div.set-time > p');
+const { DateTime } = this.luxon;
+
+const myListPage = () => {
+  contactPage.style.display = 'none';
+  formInput.style.display = 'none';
+  myBookList.style.display = 'block';
+};
+navListBtn.addEventListener('click', myListPage);
```
Comment (Dipeshtwis) [inline index.js]: - [ ] The Booklist page also does not follow the design. Please make sure to keep the design same as this is the hard requirement of this milestone.

### Your design

Screenshot from 2021-11-25 19-52-58

### Original design

Screenshot from 2021-11-25 19-53-16

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @kingqabil and @cwaku, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi @kingqabil & @cwaku 🥇, 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**